go-controller/pkg/ovn/ovn.go

```diff
@@ -209,6 +210,18 @@ type Controller struct {
 	retryPolicyChan chan struct{}
 
 	metricsRecorder *metrics.ControlPlaneRecorder
+
+	// nodeController
+	nodes       listers.NodeLister
+	nodesSynced cache.InformerSynced
+	nodesQueue  workqueue.RateLimitingInterface
+	nodesCache  sync.Map
+
+	// namespaceController
+	namespacesLister listers.NamespaceLister
+	namespacesSynced cache.InformerSynced
+	namespacesQueue  workqueue.RateLimitingInterface
+	namespacesCache  sync.Map
```

Comment:
we already have 
```
	// Info about known namespaces. You must use oc.getNamespaceLocked() or
	// oc.waitForNamespaceLocked() to read this map, and oc.createNamespaceLocked()
	// or oc.deleteNamespaceLocked() to modify it. namespacesMutex is only held
	// from inside those functions.
	namespaces      map[string]*namespaceInfo
	namespacesMutex sync.Mutex
```

to act as namespace cache which has some pretty heavy locking and complex code around it in OVNK. I don't think we want to add another namespace cache here.